### PR TITLE
Added Microsoft.Kusto version 2019-09-07

### PR DESCRIPTION
**Pull request by saguiitay**

### CODE REVIEW

**Comment by saguiitay** [comment]
Please DO NOT MERGE until discussing with me. Thanks.

File: schemas/2019-09-07/Microsoft.Kusto.json [hunk 1]
```diff
@@ -0,0 +1,621 @@
+﻿{
```
Comment (anthony-c-martin) [inline schemas/2019-09-07/Microsoft.Kusto.json]: Please see https://github.com/Azure/azure-resource-manager-schemas#submitting-a-pr

You're missing adding a link to all the root schemas - without this, customers won't be able to use this resource in a template.

Comment (saguiitay) [reply schemas/2019-09-07/Microsoft.Kusto.json]: Thanks. Should be resolved.

File: schemas/2019-09-07/Microsoft.Kusto.json [hunk 2]
```diff
@@ -0,0 +1,621 @@
...
+      "description": "An array represents the availability zones of the cluster.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "ClusterProperties": {
+      "type": "object",
+      "properties": {
+        "trustedExternalTenants": {
```
Comment (anthony-c-martin) [inline schemas/2019-09-07/Microsoft.Kusto.json]: Missing expression oneOf

File: schemas/2019-09-07/Microsoft.Kusto.json [hunk 3]
```diff
@@ -0,0 +1,621 @@
...
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TrustedExternalTenant"
+          },
+          "description": "The cluster's external tenants."
+        },
+        "optimizedAutoscale": {
```
Comment (anthony-c-martin) [inline schemas/2019-09-07/Microsoft.Kusto.json]: Missing expression oneOf

File: schemas/2019-09-07/Microsoft.Kusto.json [hunk 4]
```diff
@@ -0,0 +1,621 @@
...
+          "type": "object",
+          "$ref": "#/definitions/OptimizedAutoscale",
+          "description": "Optimized auto scale definition."
+        },
+        "enableDiskEncryption": {
```
Comment (anthony-c-martin) [inline schemas/2019-09-07/Microsoft.Kusto.json]: Missing expression oneOf

File: schemas/2019-09-07/Microsoft.Kusto.json [hunk 5]
```diff
@@ -0,0 +1,621 @@
...
+    "OptimizedAutoscale": {
+      "type": "object",
+      "required": [
+        "version",
+        "isEnabled",
+        "minimum",
+        "maximum"
+      ],
+      "properties": {
+        "version": {
```
Comment (anthony-c-martin) [inline schemas/2019-09-07/Microsoft.Kusto.json]: All these properties are also missing expression declarations

**Pull request merged**